app/js/actions/createOrder.js

```diff
@@ -3,9 +3,9 @@ import axiosInstance from '../config';
 
 import { SAVE_DRAFT_LAB_ORDER } from './actionTypes';
 
-const createLabOrder = labOrderData => ({
+const createOrder = labOrderData => ({
```

Comment:
the parameter name here should also change to a more befitting variable name.

Reply:
Resolved.